controllers/schools.js

```diff
@@ -3,27 +3,20 @@ const express = require('express');
 const router = express.Router();
 const api = require('../api');
 
-// schools
-
+// This route is only used for the external invite form. That's why the API call is specific for that.
 router.get('/', async (req, res, next) => {
 	const params = {
 		qs: {
-			$limit: req.query.$limit,
-			federalState: req.query.federalState,
-			$sort: 'name',
+			federalStateId: req.query.federalState,
```

Comment:
Do we need no pagination anymore and the filter to remove the already selected school?

Reply:
I removed pagination and the filter because I made the backend call specific for the external invite form. See the comment on the route. 

Pagination wasn't used in this use case before. I guess it was just there to keep the route generic.

The user's school is now filtered out in the backend.

Reply:
More precise: The user's school is now ALWAYS filtered out in the backend, not depending on a parameter.